### DISCUSSION TITLE
NLP nummers laten filteren

### DISCUSSION THREAD

**Discussion by melvin2204** [lmeulen/PrivacyFilter#25] [NLP]
De NLP kan ook voluit geschreven getallen herkennen. Moeten deze ook gefilterd worden?

Bijvoorbeeld: `Ik heb vijf Raspberry Pi's` zou dan `Ik heb <GETAL> <ORG>` worden.

Dit is de huidige output van het NLP filter:
```
Ik heb vijf Raspberry Pi's
###################
0 5 Ik VNW|pers|pron|nomin|vol|1|ev PRON  False 
1 5 heb WW|pv|tgw|ev AUX  False 
2 5 vijf TW|hoofd|prenom|stan NUM CARDINAL False 
3 5 Raspberry SPEC|deeleigen PROPN ORG True 
4 5 Pi's SPEC|deeleigen PROPN ORG True Raspberry 
Ik heb vijf Raspberry Pi's~Ik heb vijf <ORG>
```

**Comment by lmeulen** [comment]
Ja, als het kan moeten we deze uit laten filteren.

**Reply by melvin2204** [reply]
Ik ga kijken naar een mogelijke oplossing in de nlp_filter branch

**Comment by lmeulen** [comment]
Dit is inmiddels geïmplementeerd in het NLP filter
